src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -45,10 +45,12 @@ public SignatureData(SignatureHeader header, ImmutableArray<ParameterSymbol> par
         // This type is used to compact many different bits of information efficiently.
         private struct PackedFlags
         {
-            // We currently pack everything into a 32-bit int with the following layout:
+            // We currently pack everything into a 64-bit long with the following layout:
```

Comment:
>64-bit long [](http://example.com/codeflow?start=51&length=11)

It seems like a fair amount of change here and duplicating code in `EnumUtilities` and `ThreadSafeFlagOperations`. Would it be less change (especially if we think we'll find an alternative to adding bits in the future), to add an `int _bits2;` field for the new bits instead?

Reply:
I'll take a stab at this as an `int` for now. If anyone can advise on how this could be synchronized with a `byte`, for example, I can try that as well.

Related to https://github.com/dotnet/roslyn/pull/67432#discussion_r1153598816